lib/compilation.sh

```diff
@@ -474,6 +474,19 @@ CUSTOM_KERNEL_CONFIG
 
 		[[ ${PIPESTATUS[0]} -ne 0 ]] && exit_with_error "Error kernel menuconfig failed"
 
+		# The local version in the configuration file must be empty.
+		# Use the ADDITIONAL_LOCALVERSION variable to
+		# add to the local version, if required.
+		LOCALVERSION="$(awk -F'\"' '/^CONFIG_LOCALVERSION/{print $2}' .config)"
+		if [ "$LOCALVERSION" != "" ]; then
+			display_alert "CONFIG_LOCALVERSION=" "$LOCALVERSION" "wrn"
+			display_alert "Fix it." "" "info"
```

Comment:
If this supposed to be warning, then:

display_alert "Fix it." "" "wrn"

Reply:
Ambiguity of translation.
```
display_alert "Exists CONFIG_LOCALVERSION=" "$LOCALVERSION" "wrn"
display_alert "This variable should be empty. Fixing it." "info"
display_alert "Use the ADDITIONAL_LOCALVERSION variable for this." "info"
```

Reply:
Aha, warning is already in 1st line.

Reply:
I'm looking at how it's done in other distributions. openSUSE, for example, writes all the information to "CONFIG_LOCALVERSION=" with the sed editor. But their build scripts are so configured historically. I suggest not writing anything to this kernel configuration variable.